test/runtests.py

```diff
@@ -356,16 +356,9 @@ def timeout_func(timeout_data):
             if baseline:
                 # perform baseline comparison
                 baseline = self._check_file(working_path, baseline)
-                expected_output = None
-                with open(baseline, 'r') as bs_file:
-                    baseline_output = bs_file.read()
-
-                # Cleanup carriage return
-                # todo: remove carriage return at the end of the line
-                #       or better fix ch to output same on all platforms
-                expected_output = re.sub('[\r]+\n', '\n', baseline_output)
-
-                # todo: implement wild cards support
+                with open(baseline, 'rb') as bs_file:
+                    expected_output = bs_file.read().decode('utf-8') \
+                                                    .replace('\r','')
```

Comment:
> Could you please elaborate? I didn't understand how it's different now ?

The difference is from changing open file mode `r` to `rb`. (In mode `r` python may do end-of-line conversions.) For the same file we get `\r\n\n` and `\r\r\n` respectively in these two modes, resulting in `\n\n` and `\n` after our `replace`.